docs/index.asciidoc

```diff
@@ -131,31 +133,54 @@ content-type as the event message.
 
 
 [id="plugins-{type}s-{plugin}-port"]
-===== `port` 
+===== `port`
 
   * Value type is <<number,number>>
   * There is no default value for this setting.
 
 
 
 [id="plugins-{type}s-{plugin}-secure"]
-===== `secure` 
+===== `secure`
 
   * Value type is <<boolean,boolean>>
   * Default value is `true`
 
 
 
+[id="plugins-{type}s-{plugin}-sincedb_path"]
+===== `sincedb_path`
+
+  * Value type is <<string,string>>
+  * There is no default value for this setting.
+
+Path of the sincedb database file (keeps track of the last processed IMAP
+uid) that will be written to disk. The default will write sincedb files to
+`<path.data>/plugins/inputs/imap`
+NOTE: it must be a file path and not a directory path.
+
 [id="plugins-{type}s-{plugin}-strip_attachments"]
-===== `strip_attachments` 
+===== `strip_attachments`
 
   * Value type is <<boolean,boolean>>
   * Default value is `false`
 
 
 
+[id="plugins-{type}s-{plugin}-uid_tracking"]
+===== `uid_tracking`
+
+  * Value type is <<boolean,boolean>>
+  * Default value is `false`
+
```

Comment:
Can we have a small explanation as to what `uid_tracking` is, and a description of the differences in behaviour  between having this flag set and not?

Reply:
Fixed in e0a12bb